bin/wdq-batch

```diff
@@ -66,21 +66,30 @@ parser.add_argument('-o', '--output-dir', default=os.getcwd(),
 parser.add_argument(
     '-f', '--config-file',
     help='path to configuration file to use, default: '
-         '~detchar/etc/omega/{epoch}/{OBS}-{IFO}_R-selected.txt')
-parser.add_argument(
-    '-c', '--cache-file',
-    help='path to data cache file, if not given, data locations '
-         'are found using the datafind server, must be in LAL cache format')
+         '~detchar/etc/omega/{epoch}/{OBS}-{IFO}_R-selected.{txt/ini}')
 parser.add_argument('-q', '--wdq', default=WDQ, required=WDQ is None,
                     help='path to wdq executable')
 parser.add_argument('-w', '--wpipeline', default=omega.WPIPELINE,
                     required=omega.WPIPELINE is None,
-                    help='path to wpipeline binary')
+                    help='path to Matlab wpipeline binary or the string '
+                         '"gwdetchar-omega", default: %(default)s')
+parser.add_argument('--colormap', default='parula',
```

Comment:
@alurban, I think this should have a default of `None` now, and then lookup the default for the given impementation, something like:

```python
DEFAULT_COLORMAP = {
    'gwdetchar-omega': 'viridis',
    'wpipeline': 'parula',
}
```

I haven't tested, but I presume that right now if I specify `--wpipeline=gwdetchar-omega` then the underlying call will fail because matplotlib doesn't recognise `parula`.